mitosheet/css/elements/SearchBar.css

```diff
@@ -1,6 +1,6 @@
 .mito-search-bar {
     position: absolute;
-    top: 72px;
+    top: 117px;
```

Comment:
This height has to be dynamic somehow, I think. Cus when the toolbar is closed it's at the wrong height. Wanna at least open an issue so we can come back to it in the future?

Reply:
Ah ok yes

Reply:
I'll add a note in the spec